packages/elements/src/space/index.js

```diff
@@ -0,0 +1,39 @@
+const SPACING_BASE = 8;
+
+/**
+ * Converts pixels to rem.
+ *
+ * @param {Number} [px=0]
+ */
+const pxToRem = px => px / 16;
+
+/**
+ * Returns a rem-based spacing.
+ *
+ * @param {Number} [step=0]
+ *    The number of steps up or down.
+ *    Defaults to the base spacing - 8px (0.5rem).
+ */
+const spacing = (step) => {
+  if (!step) return `${pxToRem(SPACING_BASE)}rem`;
+
+  if (step < 0) return `${pxToRem(SPACING_BASE / (step * -2))}rem`;
+
+  return `${pxToRem(step * SPACING_BASE)}rem`;
+};
```

Comment:
If the output value is in `rem` we don't need a middle step that uses a `px` value 
Don't support negative values just to match a label like `minus1` - 
`step * SPACING_BASE` is very easy to understand and work for these values `spacing(0.5)` or `spacing(0.25)`
while `SPACING_BASE / (step * -2)` isn't so straightforward to decipher 

Since the function is only not exported we can defined it like so and call it with the correct parameters

```js
const SPACING_BASE = 0.5;
const spacing = (step = 1) => `${step * SPACING_BASE}rem`;
```